src/main/resources/processes/MPAM_TRIAGE.bpmn

```diff
@@ -361,6 +363,58 @@
+      <bpmn:incoming>SequenceFlow_1d5y4cw</bpmn:incoming>
+      <bpmn:incoming>SequenceFlow_0cc989h</bpmn:incoming>
+      <bpmn:outgoing>SequenceFlow_0oh8e3b</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:exclusiveGateway id="ExclusiveGateway_0mmxl4e" name="TriageOutcome?">
+      <bpmn:incoming>SequenceFlow_0hgwrhp</bpmn:incoming>
+      <bpmn:outgoing>SequenceFlow_1qyqxc7</bpmn:outgoing>
+      <bpmn:outgoing>SequenceFlow_094r7km</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:userTask id="UserTask_18mxbbm" name="Request Campaign">
+      <bpmn:incoming>SequenceFlow_0oh8e3b</bpmn:incoming>
+      <bpmn:outgoing>SequenceFlow_05730uo</bpmn:outgoing>
+    </bpmn:userTask>
+    <bpmn:exclusiveGateway id="ExclusiveGateway_0sd09rb" name="Direction Check">
+      <bpmn:incoming>SequenceFlow_05730uo</bpmn:incoming>
+      <bpmn:outgoing>SequenceFlow_0jtqv93</bpmn:outgoing>
+      <bpmn:outgoing>SequenceFlow_09fnnt9</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:exclusiveGateway id="ExclusiveGateway_13i24qr">
+      <bpmn:incoming>SequenceFlow_0jtqv93</bpmn:incoming>
+      <bpmn:outgoing>SequenceFlow_1d5y4cw</bpmn:outgoing>
+      <bpmn:outgoing>SequenceFlow_1legare</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:sequenceFlow id="SequenceFlow_1d5y4cw" sourceRef="ExclusiveGateway_13i24qr" targetRef="ServiceTask_0011raj">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${valid == false}</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="SequenceFlow_0oh8e3b" sourceRef="ServiceTask_0011raj" targetRef="UserTask_18mxbbm" />
+    <bpmn:sequenceFlow id="SequenceFlow_05730uo" sourceRef="UserTask_18mxbbm" targetRef="ExclusiveGateway_0sd09rb" />
+    <bpmn:sequenceFlow id="SequenceFlow_0jtqv93" sourceRef="ExclusiveGateway_0sd09rb" targetRef="ExclusiveGateway_13i24qr">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${DIRECTION == "FORWARD"}</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="SequenceFlow_1qyqxc7" name="else" sourceRef="ExclusiveGateway_0mmxl4e" targetRef="ExclusiveGateway_1ivf0n0">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${TriageOutcome != "PutOnCampaign"}</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="SequenceFlow_094r7km" name="PutOnCampaign" sourceRef="ExclusiveGateway_0mmxl4e" targetRef="ServiceTask_1uwrjca">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${TriageOutcome == "PutOnCampaign"}</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="SequenceFlow_09fnnt9" sourceRef="ExclusiveGateway_0sd09rb" targetRef="ServiceTask_05nlmnl">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${DIRECTION == "BACKWARD"}</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="SequenceFlow_1legare" sourceRef="ExclusiveGateway_13i24qr" targetRef="ServiceTask_1t6qiuf" />
```

Comment:
This sequence flow seems to be missing a condition.

Reply:
Ooops, will fix it on my next commit to workflow (haven't noticed the warning). Luckily this has no impact on the flow.